data/json/monsters/feral_humans.json

```
@@ -74,5 +74,125 @@
     "melee_dice": 3,
     "melee_dice_sides": 8,
     "death_drops": "feral_humans_death_drops_axe"
+  },
+  {
+    "id": "mon_feral_scientist_scalpel",
+    "type": "MONSTER",
+    "name": { "str": "mad scientist" },
+    "description": "A researcher who has stared too long into the abyss, and now shambles about muttering nonsense under their breath.  For some reason, the zombies don't seem to mind that this person is obviously still alive.  They're clutching a tiny but razor-sharp blade in one shaky hand.",
+    "default_faction": "science",
+    "looks_like": "chud",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 1,
+    "melee_dice_sides": 3,
+    "melee_cut": 2,
+    "dodge": 1,
+    "harvest": "CBM_SCI_FERAL",
```

```suggestion
    "harvest": "CBM_SCI_FERAL",
    "path_settings": { "max_dist": 10 },
```
Their zombies have max_dist 5.